bms/bms_app/static/bms_app/css/create_project.css

```diff
@@ -63,33 +79,39 @@
 }
 
 .remove{
-    margin-right: 10px;
+	margin-right: 10px;
+	vertical-align: top;
 }
 
 .glyphicon-remove{
 	font-size: 14px;
-	top: 0px;
+	top: 0;
```

Comment:
top: 0; and right: 0; properties will work only with position property, without position property they don't make any difference in CSS